### ISSUE TITLE
Inquiry about target_train_dataset and target_test_dataset

### ISSUE THREAD

**Issue by 1320harry** [JoonHo-Jang/UADAL#1]
Hello UADAL authors! Congratulations on being accepted to Neurips 2022, and thank you for sharing your excellent codes!

I have one question about how you split the target domain data.

According to your code, you divided target domain data into 3 splits ("target train set", "target validation set", and "target test set"), and used the "target train set" during training (with "source train set") while using the "target test set" at inference time (for calculating model accuracy).

However, when I printed all 3 splits of target domain data, I found that they were all the same. I think you missed the codes that split the target domain data. (If I misunderstood your codes, please let me know.)

Would you please add the code that splits target domain data?

**Comment by accocca** [comment]
> Hello UADAL authors! Congratulations on being accepted to Neurips 2022, and thank you for sharing your excellent codes!
> 
> I have one question about how you split the target domain data.
> 
> According to your code, you divided target domain data into 3 splits ("target train set", "target validation set", and "target test set"), and used the "target train set" during training (with "source train set") while using the "target test set" at inference time (for calculating model accuracy).
> 
> However, when I printed all 3 splits of target domain data, I found that they were all the same. I think you missed the codes that split the target domain data. (If I misunderstood your codes, please let me know.)
> 
> Would you please add the code that splits target domain data?

Having the same issues here, this code seems to be missing a split of the dataset.

**Comment by wyg0407** [comment]
关于数据集划分的问题我和您有相同的疑问，请问您解决了吗
